Simplenote.xcodeproj/project.pbxproj

```diff
@@ -2716,7 +2716,7 @@
 				PRODUCT_NAME = Simplenote;
 				PROVISIONING_PROFILE = "";
 				"PROVISIONING_PROFILE[sdk=macosx*]" = "";
-				PROVISIONING_PROFILE_SPECIFIER = "Simplenote Mac Production Profile";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
Removed the hardcode values so we can use whatever `match` fetches for us.

Reply:
This turned out to be a mistake! For some reason, it resulted in the sandboxing entitlement not being ported over to the `.pkg` which failed the upload to App Store Connect.